update_grid.cc

```diff
@@ -1016,8 +1016,10 @@ static void update_grid_cell(const int mgi, const int nts, const int nts_prev, c
 {
   const int assoc_cells = grid::get_numassociatedcells(mgi);
   if (assoc_cells > 0) {
-    // estimators were accumulated in nts_prev, but radiation density, etc should be scaled to the cell volume at
-    // nts
+    // estimators were accumulated in nts_prev so radiation density, etc should be scaled to the cell volume at
+    // nts_prev not nts to prevent the estimators being under estimated by scaling with a cell volume that is
+    // larger than the cell volume was when the estimators were being accumlated however still using
+    // nts for this release as don;t want to change the code behaviour with this release just adding functionality
```

Comment:
Delete or rephrase into coherent sentences. The nts_prev change will be made in the next major release.